### PR TITLE
Document api delete post

### PR DESCRIPTION
**Pull request by at-trampham**
## Description
Document api delete post
## Reference Requirements
- Product Backlog ID: Non PBI
- Task ID: Non TI

## Reference Isuees
Non

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
NonSql

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: docs/api-references/post.md [hunk 1]
```diff
@@ -551,3 +551,31 @@ Create new post
...
+| Field | Type | Description |
+|---|---|---|
+| meta | Object | Object meta |
+| status | String | Status result |
+| code | Number | HTTP status code |
+```json
+{
+    "meta": {
+        "status": "Successfully",
+        "code": 200
```
Comment (at-minhdao) [inline docs/api-references/post.md]: xem lại thử là trả về `200` hay `204`

**Review by at-minhdao — COMMENTED** [review]
check cmt

File: docs/api-references/post.md (hunk 1 above)
Comment (at-trampham) [reply docs/api-references/post.md]: trả về `200` nhé

**Comment by at-minhdao** [comment]
LGTM!

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**